pytest_extra_requirements.txt

```diff
@@ -8,4 +8,3 @@ mock ; python_version < "3.0"
 # ansible and dependencies for all supported platforms
 ansible ; python_version > "2.6"
 idna<2.8 ; python_version < "2.7"
-PyYAML<5.1 ; python_version < "2.7"
```

Comment:
IMHO, it is better to explain why it is not needed anymore in the commit message.

Reply:
yes, you are right. I don't know why it is not needed anymore or why it was added.

Reply:
@richm do you know/remember why  PyYAML is there?

Reply:
> @richm do you know/remember why PyYAML is there?

No, I do not remember